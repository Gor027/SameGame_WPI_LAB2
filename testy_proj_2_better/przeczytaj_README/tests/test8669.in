`x `x`  `    `   x xx`  xx`x`x ``xx`
`xx`  `` xx  ``x``x`xx x``` x ` x`x ```x `x`x  x
   x`  `x  x `x`   ``xx x```x xx`x x    x` xx `  ``x x   x` 
  `` xx`  `x `x`  ``x`x`x x`x x ``xx ```` ` x `  xx x x` x    ``x    `xxx x 
 `x` x ` xxx  xx``x xx    `  xx` xx`x  x
 xx x`  ` x ```` xxxx` x`xxx x `xxxx`x x  ```
`xx xxx x` ` `x` x` `xx`   `xx `x` ``xx    ``x  xx  `  x`  `xx x  ````x``x` 
` ` `xxx x   x x   `xxx xxx` x`x`xxx`x``x
x x`xx``x  x`xx `  xx``xx`x` ```x`xx`  ````x``xxxxx`x xxxx `  x ``  ``  xxx` 
 x`xx``` ` xx`x`x`  x`xxx ```x xx xxx``  x  `xx``xx 
