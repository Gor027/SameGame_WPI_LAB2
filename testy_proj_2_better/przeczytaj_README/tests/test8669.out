                    x`
             `      xx    `     `x `
     `       ``x    `x  x `x   `x`xx
  x`xx `     `x`   xx`  ` `` x x`xxx`       x
  ```x``   x `x` x `xxxxxxx``x````x`x` x    x
  xxx`x``x x `xx `x`x`x`x``xxxx`xxxx````   `x    `
` x`xxx`xxxx ````````xxx``x`xxx`xxx``x`xx`xx` `  x`
`xx``xxx`````xx``xxxxxx`xxx`xxxx``xx`xxx`````x`xxxx x  ` x           `  x x
`x``xx``xxxx`xxx`xx`x``xxxx`x```xxxx`x``x`````xxxxxx` xx`x `  ``x   ``xx`x`
```xx```x`xxx`x`x``xx`xxx````x`xx`xxx````x`x`xx``xx`xxxxxx``xxxx``````x`xxx`
